.expo/README.md

```diff
@@ -0,0 +1,17 @@
+> Why do I have a folder named ".expo" in my project?
+
+The ".expo" folder is created when an Expo project is started using "expo start" command.
```

Comment:
According to this file, the .expo folder should not be committed to Git. Make sure that `.expo/` in the `.gitignore` file, and then type `git rm expo/` or something like that.

Reply:
Sorry, I completely missed this. Just fixed it!